driver-async/src/test/functional/com/mongodb/async/client/JsonPoweredCrudTestHelper.java

```diff
@@ -164,10 +165,8 @@ private void appendInitCapToBuilder(final StringBuilder builder, final String ob
     <T> T futureResult(final FutureResultCallback<T> callback) {
```

Comment:
I changed FutureResultCallback.get to timeout after 5 mins - having it block forever seemed needless.